authorization/src/main/java/uk/gov/di/ipv/cri/common/api/handler/AuthorizationHandler.java

```diff
@@ -68,9 +70,21 @@ public APIGatewayProxyResponseEvent handleRequest(
             SessionItem sessionItem = sessionService.getSession(sessionId);
             eventProbe
                     .addJourneyIdToLoggingContext(sessionItem.getClientSessionId())
-                    .log(Level.INFO, "found session");
+                    .log(INFO, "found session");
             // validate
             authorizationValidatorService.validate(authenticationRequest, sessionItem);
+
+            // Return access denied if there is no authcode found
+            if (StringUtils.isBlank(sessionItem.getAuthorizationCode())) {
```

Comment:
Should this code not leave inside the `authorizationValidatorService` which then throws the error that can be caught and handled as an accessDenied error

Reply:
I think there is a conflict between the behaviour of all the CRI's for what a user reaching /authorize with no authcode means in the processing flow.

For FraudCRI user don't get forwarded to authorise without an auth-code having been generated (fraud returns internal server), the oauth access_denied should not break anything if one did but it would technically be an error for that CRI.

In PassportCRI as it is live (with its own version of /authorize lambda) users can reach /authorize with-out an authcode if they decide not to fill in the form and click prove-another-way which results in the OAuth access denied - it not an error pathway as such.

DrivingLicenceCRI prove-another-way has been modelled on passports way of handling prove-another-way, but is using the common-lambdas /authorize lambda, so reaching /authorize without an authcode would not be an error also.